### PR TITLE
br: modify vbk failed when all backups are complete or failed

### LINKED ISSUE #5725: ebs br: don't mark volumebackup failed too early
## Bug Report

**What version of Kubernetes are you using?**

**What version of TiDB Operator are you using?**

**What storage classes exist in the Kubernetes cluster and what are used for PD/TiKV pods?**

**What's the status of the TiDB cluster pods?**

**What did you do?**

Schedule routine volume backups

**What did you expect to see?**
No parallel volume backups run

**What did you see instead?**
Parallel volume backups are active when last one failed at some backup member.

### PR DESCRIPTION
**Pull request by WangLe1321**
### What problem does this PR solve?

Closes #5725 

### What is changed and how does it work?

only set volume backup is done (failed or complete) after all data planes are done at waiting snapshot creation time whose during is long.

### Code changes

- [x] Has Go code change
- [ ] Has CI related scripts change

### Tests

- [x] Unit test 
- [ ] E2E test 
- [ ] Manual test 
- [ ] No code 

### Side effects

- [ ] Breaking backward compatibility 
- [ ] Other side effects: 

### Related changes

- [ ] Need to cherry-pick to the release branch 
- [ ] Need to update the documentation 

### Release Notes

Please refer to [Release Notes Language Style Guide](https://github.com/pingcap/tidb-operator/blob/master/docs/release-note-guide.md) before writing the release note.

```release-note

```

### CODE REVIEW

File: pkg/fedvolumebackup/backup/backup_manager.go [hunk 1]
```diff
@@ -425,14 +426,18 @@ func (bm *backupManager) teardownVolumeBackup(ctx context.Context, volumeBackup
 
 func (bm *backupManager) waitVolumeBackupComplete(ctx context.Context, volumeBackup *v1alpha1.VolumeBackup, backupMembers []*volumeBackupMember) error {
 	isBackupRunning := false
+	var failedBackups []*volumeBackupMember
 	for _, backupMember := range backupMembers {
-		if pingcapv1alpha1.IsVolumeBackupInitializeFailed(backupMember.backup) || pingcapv1alpha1.IsBackupFailed(backupMember.backup) {
+		if pingcapv1alpha1.IsVolumeBackupInitializeFailed(backupMember.backup) {
```
Comment (BornChanger) [inline pkg/fedvolumebackup/backup/backup_manager.go:431]: are you sure only when all members have complete initialize phase can start real work on aws invocation?

Comment (WangLe1321) [reply pkg/fedvolumebackup/backup/backup_manager.go:431]: Yes, we execute initialization at first to pause GC and PD schedulers. After initialize successfully, we execute the actual backup phase. So if initialize failed, no volume snapshot will be created.

**Review by BornChanger — APPROVED** [review]

**Pull request merged**